### PR TITLE
3 get categories

### PR DESCRIPTION
**Pull request by DanMossDev**
Thank you!

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    db.end()
+})
+
+describe('/api/categories', () => {
+    describe('GET requests', () => {
+        describe('GET /api/categories', () => {
+            test('happy path', () => {
+                return request(app).get('/api/categories').expect(200).then(({body}) => {
+                    expect(Array.isArray(body)).toBe(true)
+                    body.forEach(object => {
```
Comment (emilyb93) [inline __tests__/app.test.js:22]: Make sure any time we do a test on an array, we also make sure we are checking the correct length, a forEach test on an empty array will always pass so its important we check that we are at least not testing an empty array

**Review by emilyb93 — CHANGES_REQUESTED** [review]
Just a change to the tests and then this is good to go!

File: __tests__/app.test.js (hunk 1 above)
Comment (DanMossDev) [reply __tests__/app.test.js:22]: Thanks for your help! Shall I just merge it with those changes or submit another request?

Comment (emilyb93) [reply __tests__/app.test.js:22]: This is good to go!

**Review by emilyb93 — APPROVED** [review]
